deployment/windows/jellyfin.nsi

```diff
@@ -0,0 +1,202 @@
+; Shows a lot of debug information while compiling
+; This can be removed once stable.
+!verbose 4
+;--------------------------------
+;Include Modern UI
+
+    !include "MUI2.nsh"
+    Var _JELLYFINVERSION_
+    Var _DEFAULTEMBYDATADIR_
+    Var _JELLYFINDATADIR_
+    Var _SERVICEINSTALLED_
+;--------------------------------
+;General
+
+; Align installer version with jellyfin.dll version
+    !getdllversion "$%InstallLocation%\jellyfin.dll" ver_
+    !echo "jellyfin.dll version is ${ver_1}.${ver_2}.${ver_3}.${ver_4}" ;!echo will print it while building
+
+    Name "Jellyfin ${ver_1}.${ver_2}.${ver_3}.${ver_4}"
+    OutFile "Jellyfin.Installer.${ver_1}.${ver_2}.${ver_3}.${ver_4}.exe"
+    BrandingText "Jellyfin ${ver_1}.${ver_2}.${ver_3}.${ver_4} Installer"
+
+; installer attributes
+    VIProductVersion "${ver_1}.${ver_2}.${ver_3}.${ver_4}"
+    VIFileVersion "${ver_1}.${ver_2}.${ver_3}.${ver_4}"
+    VIAddVersionKey "ProductName" "Jellyfin"
+    VIAddVersionKey "FileVersion" "${ver_1}.${ver_2}.${ver_3}.${ver_4}"
+  
+  ;Default installation folder
+    InstallDir "$APPDATA\Jellyfin"
+  
+  ;Get installation folder from registry if available
+    InstallDirRegKey HKLM "Software\Jellyfin" "InstallLocation"
+  
+  ;Request application privileges for Windows Vista
+    RequestExecutionLevel admin
+    CRCCheck on	    
+    !define MUI_ABORTWARNING
+
+;--------------------------------
+;Pages
+
+;TODO
+;find a license to displayed before installer is started
```

Comment:
This should be the LICENSE from the main repo, and probably also misc licenses of other components bundled in the installer.

Reply:
Agree, I had left it as TODO as there is currently no license copied to installation. 
I've now updated build script to copy it and then included it here to show before installation.